src/CMakeLists.txt

```diff
@@ -202,6 +195,10 @@ set(utils_files
     nupic/utils/SdrMetrics.hpp
     nupic/utils/MurmurHash3.hpp
     nupic/utils/MurmurHash3.cpp
+    nupic/utils/StlIo.cpp
+    nupic/utils/StlIo.hpp
```

Comment:
nit, maybe for separate PR but StlIO can be removed too. Only included in SP/TM tests (so we could live w/o it)

Reply:
Tracked in issue #175.